src/I18nProvider.js

```diff
@@ -56,11 +56,18 @@ function objectInterpolation(obj, query) {
   return obj
 }
 
+function missingKeyLogger({ namespace, i18nKey }) {
+  console.warn(
```

Comment:
@giovannigiordano it would make sense to do nothing by default in production?
```suggestion
  if (process.env.NODE_ENV === 'production') return
  console.warn(
```

Reply:
It's what asked in the issue to log missing keys, I can add it as an option instead of excluding it for production